src/GPT2-Simple.ipynb

```diff
@@ -1 +1,991 @@
+      "Fetching model.ckpt.data-00000-of-00001: 1.42Git [00:15, 92.6Mit/s]                                 \n",
+      "Fetching model.ckpt.index: 1.05Mit [00:00, 265Mit/s]                                                \n",
+      "Fetching model.ckpt.meta: 1.05Mit [00:00, 57.2Mit/s]                                                \n",
+      "Fetching vocab.bpe: 1.05Mit [00:00, 135Mit/s]                                                       \n"
+     ]
+    }
+   ],
+   "source": [
+    "gpt2.download_gpt2(model_name=\"355M\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "6OFnPCLADfll"
+   },
+   "outputs": [],
+   "source": [
+    "file_name = \"../input/LesMiserables.txt\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "LdpZQXknFNY3"
+   },
+   "source": [
+    "## Finetune GPT-2\n",
+    "\n",
+    "The next cell will start the actual finetuning of GPT-2. It creates a persistent TensorFlow session which stores the training config, then runs the training for the specified number of `steps`. (to have the finetuning run indefinitely, set `steps = -1`)\n",
+    "\n",
+    "The model checkpoints will be saved in `/checkpoint/run1` by default. The checkpoints are saved every 500 steps (can be changed) and when the cell is stopped.\n",
+    "\n",
+    "The training might time out after 4ish hours; make sure you end training and save the results so you don't lose them!\n",
+    "\n",
+    "**IMPORTANT NOTE:** If you want to rerun this cell, **restart the VM first** (Runtime -> Restart Runtime). You will need to rerun imports but not recopy files.\n",
+    "\n",
+    "Other optional-but-helpful parameters for `gpt2.finetune`:\n",
+    "\n",
+    "\n",
+    "*  **`restore_from`**: Set to `fresh` to start training from the base GPT-2, or set to `latest` to restart training from an existing checkpoint.\n",
+    "* **`sample_every`**: Number of steps to print example output\n",
+    "* **`print_every`**: Number of steps to print training progress.\n",
+    "* **`learning_rate`**:  Learning rate for the training. (default `1e-4`, can lower to `1e-5` if you have <1MB input data)\n",
+    "*  **`run_name`**: subfolder within `checkpoint` to save the model. This is useful if you want to work with multiple models (will also need to specify  `run_name` when loading the model)\n",
+    "* **`overwrite`**: Set to `True` if you want to continue finetuning an existing model (w/ `restore_from='latest'`) without creating duplicate copies. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**EXTRA IMPORTANT NOTE**: If running the notebook in Google Colab, it is advised to mount your Drive to be able to save checkpoints and re-start from where you stopped in case the notebook crashes or times out. "
```

Comment:
First change: explaining the importance of saving checkpoints and presenting the possibility of doing it in Drive